hop/__main__.py

```diff
@@ -8,22 +8,51 @@
 from . import version
 
 
-def append_subparser(subparser, cmd, func):
+def append_subparser(subparser, cmd, func, formatter=False):
```

Comment:
A boolean for `formatter` seems slightly odd. I would have expected it to be a formatter class, so you pass in `formatter=_SubcommandHelpFormatter`.

Reply:
That's a good idea, I added this in the last batch of commits.